eng/pipelines/create-doc-index.yml

```diff
@@ -27,10 +27,17 @@ jobs:
         inputs:
           versionSpec: $(PythonVersion)
 
+      - template: eng/pipelines/templates/scripts/replace-relative-links.yml@azure-sdk-tools
+        parameters:
+          TargetFolder: $(Build.SourcesDirectory)
```

Comment:
Out of curiosity some of the other repo's have passed `.` here in the context of these parameters which one is preferred?

Reply:
I'm actually validating the build results, and I'm almost positive I'll need to add the `.` explicitly. Will get back to you on this.

Reply:
Definitely necessary. Update pushed.

![image](https://user-images.githubusercontent.com/45376673/71853023-1476ad80-308f-11ea-8f28-386633c75be6.png)

Reply:
@weshaggard the java repo ran into this issue back in [December ](https://github.com/Azure/azure-sdk-for-java/pull/6925/files). On windows, it doesn't escape the backslashes correctly so we end up getting that messed up directory.